TUTORIAL.md

```diff
@@ -0,0 +1,384 @@
+
+First, define where CesiumJS is. This tutorial uses the source code, so webpack can include individual models and trace the dependencies. Alternatively, you can use the built (minified or unminified) version of CesiumJS. However, the modules are already combined and optimized, which gives us less flexibility.
+
+## Install CesiumJS
+1. Install the [Cesium](https://www.npmjs.com/package/cesium) module from npm using the command `npm install --save-dev cesium`. Install the [CopyWebpackPluigin](https://webpack.js.org/plugins/copy-webpack-plugin/) module from npm using the command `npm install copy-webpack-plugin --save-dev`.
+2. Update `sourcePrefix` to tell CesiumJS that the version of AMD webpack uses to evaluate `require` statements is not compliant with the standard `toUrl` function. In addition, add a `cesium` alias so we can reference it in our app code. After adding these changes, `webpack.config.js` should look like
+```
+// The path to the CesiumJS source code
+const cesiumSource = 'node_modules/cesium/Source';
+const cesiumWorkers = '../Build/Cesium/Workers';
+const CopywebpackPlugin = require('copy-webpack-plugin');
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    context: __dirname,
+    entry: {
+        app: './src/index.js'
+    },
+    output: {
+        filename: 'app.js',
+        path: path.resolve(__dirname, 'dist'),
+        // Needed to compile multiline strings in Cesium
+        sourcePrefix: ''
+    },
+    amd: {
+        // Enable webpack-friendly use of require in Cesium
+        toUrlUndefined: true
+    },
+    resolve: {
+        alias: {
+            // CesiumJS module name
+            cesium: path.resolve(__dirname, cesiumSource)
+        }
+    },
+    module: {
+        rules: [{
+            test: /\.css$/,
+            use: [ 'style-loader', 'css-loader' ]
+        }, {
+            test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
+            use: [ 'url-loader' ]
+        }]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html'
+        })
+    ],
+    mode: 'development',
+};
+```
+
+## Manage CesiumJS static files
+1. Now, we must make sure the static CesiumJS asset, widget, and web worker files are served and loaded correctly. Use `copy-webpack-plugin` to copy static files to the `dist` directory as part of the build process. To do this run the command `npm install --save-dev copy-webpack-plugin` and updated the plugins array in `webpack.config.js`. `webpack.config.js` should now look like
+```
```

Comment:
I noticed that `copy-webpack-plugin` isn't actually used until this step even though it's installed and referenced in the previous one. Could you move those lines to this step instead?

Reply:
Which lines should be moved?

Reply:
In step 1 of "Install CesiumJS": 

> Install the CopyWebpackPluigin module from npm using the command `npm install copy-webpack-plugin --save-dev`

Looks like this line is just repeated in this step (where it should be) so the line in Install CesiumJS can just be removed unless I'm missing something

Reply:
Ahhh I see - I just removed the extra line.